events/elections/2020/README.md

```diff
@@ -67,7 +82,7 @@ email saying something like "I accept the nomination".
 
 Once nominated, you must get the endorsement of three (3) different eligible
 voters from three (3) different employers.  If you are eligible to vote
-yourself, you count as one of the three. Endorsements from non-voting members does not count towards the final count. 
+yourself, you count as one of the three. Endorsements from non-voting members does not count towards the final count.
 
 [Eligible voters] may endorse candidates of their choosing by replying to the
 candidate's nomination email saying something like "I endorse this nominee"
```

Comment:
IIRC from last year's election, we wanted endorsers to also mention their employer along with their endorsement. Due to the minimum 3 employer restriction, the election officers had to sometimes explicitly follow up with endorsers to confirm their employer.

Having the endorser mention it removes the need for this follow up.

Reply:
added